bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleEngine.java

```diff
@@ -1345,4 +1352,65 @@ public void updated(Dictionary<String, ?> properties) throws ConfigurationExcept
         }
     }
 
+    /**
+     * The method sets default configuration values for these configuration properties which are not specified in the
+     * rule
```

Comment:
I believe that line brake (only one word of the sentence in this line) was not intendet.